index.html

```diff
@@ -3,24 +3,25 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Some Student</title>
+  <title>Dustin Walker</title>
 
   <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+  <link rel="stylesheet" href="styles/style.css">
 </head>
 
 <body>
 
-<header class="z">
+<header class="hero__container">
```

Comment:
All of the `z`s within class names can be changed to the same thing. `hero__container` should be within a `hero` element, so check that.